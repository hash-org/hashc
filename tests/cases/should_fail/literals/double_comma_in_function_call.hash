// run=fail, stage=parse

l := f(a,,);
