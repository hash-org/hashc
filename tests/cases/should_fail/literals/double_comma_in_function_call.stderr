error: expected an expression, however received a `,`
 --> $DIR/double_comma_in_function_call.hash:1:10
1 |   l := f(a,,);
  |            ^ here
